src/db/tables/onerep_scans.ts

```diff
@@ -172,9 +172,7 @@ async function getLatestOnerepScanResults(
       showRealBrokers = qaToggles.show_real_brokers;
     }
```

Comment:
In this function we can just replace this toggles logic with the feature flag logic right?